site/about.html

```diff
@@ -0,0 +1,31 @@
+<html>
+	<head>
+		<title>About Me</title> 
+		<link rel="stylesheet" href="style.css">
+
+
+	<body>
+		<nav>
+			<a href="about.html">About Me </a> |
+			<a href="portfolio.html">Portfolio</a>
+			</h3>
+		</nav>
+<h1>~Julie O.~</h1>
+
+<hr>
+
+<img src="images/TransformerWTNY.jpg" height="550" width="460" align="right"/> 
+
+	<h2> I ENJOY LIFE AS A MEDIA MAKER </h2>
+
+<p>I'm Julie O., a filmmaker based in Washington DC. I have four years of experience in the media and film world, specializing on documetaries and journalism.
+</p>
+
+<a href="https://twitter.com/">Twitter</a>
+<a href="https://www.instagram.com/julieoue/">Instagram</a>
+<a href="https://www.linkedin.com/in/julieomedia">LinkedIn</a>
+<a href="http://journalistickmedia.tumblr.com/">Film Reel</a>
+
+<hr>
+</body>
```

Comment:
A good amount of the indentation here is inconsistent. For example, the h1 tags should be at the same level as the nav tags. One thing I like to use before I submit is a code formatter to make sure things are spaced correctly: http://www.cleancss.com/html-beautify/. I'd still double check after running my code through this to make sure thing are spaced as I like them.